mappings/net/minecraft/entity/Targeter.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/unmapped/C_vgutocgh net/minecraft/entity/Targeter
```

Comment:
hm, i genuinely wonder if it's "targetter" or "targeter"

Reply:
getter is the more British way I'd think

Reply:
heck, i'm a bit british then